app/components/RepoCard/tests/index.test.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ * Tests for RepoCard
+ *
+ */
+
+import React from 'react';
+// import { fireEvent } from '@testing-library/dom'
+import { renderWithIntl } from '@utils/testUtils';
+import RepoCard from '../index';
+
+describe('<RepoCard />', () => {
```

Comment:
lets add more tests here for the different things that the RepoCard renders

Reply:
Okay!

Reply:
Updated